types/module/server/mock_module.go

```diff
@@ -0,0 +1,87 @@
+package server
+
+import (
+	"encoding/json"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/proto/tendermint/crypto"
+)
+
+/*
+* This mock module is needed to bypass the requirement of validator updates in the SDK module manager.
+* Since there are no validator updates when testing, we inject one via this mock module.
+ */
+
+var _ module.AppModule = MockModule{}
```

Comment:
not really the prettiest solution.. but the SDK module manager *requires* >0 validator updates. since we dont have any validators during testing, i put this mock module in the module manager and force it to provide a bogus validator update in its InitGenesis method.

if anyone knows a better work around please let me know lol

Reply:
The workaround works for me. https://github.com/regen-network/regen-ledger/pull/1357#discussion_r951759030